Change style

sheikhoo opened the pull request:
Just change some CSS style

![image](https://github.com/apache/tomcat/assets/11958698/44311e25-49b2-428d-89e4-5c0ace907d95)

markt-asf commented on webapps/host-manager/css/manager.css:
```diff
@@ -15,7 +15,7 @@
 
-body {
+ body {
     font-family: Tahoma, Arial, sans-serif;
```
This change appears to be unnecessary

markt-asf commented on webapps/host-manager/css/manager.css:
```diff
@@ -73,6 +79,7 @@ td.title {
     font-style    : italic;
     font-weight   : bold;
     background    : #D2A41C;
+	border-radius : 10px 10px 0 0;
```
Use 4 spaces rather than 1 tab for indents. This comment applies globally, not just to this line.

sheikhoo commented:
I removed any unnecessary spaces and utilized 4 spaces instead of 1 tab. Additionally, I updated the style of the buttons.

markt-asf commented:
Thanks for addressing the feedback. The remaining question is "What is the reason for these changes?"

sheikhoo commented:
> Thanks for addressing the feedback. The remaining question is "What is the reason for these changes?"

The reason behind these modifications is to enhance the visual appeal of the software and provide a more modern look. By removing unnecessary elements such as table borders.
I would like to redesign the frontend architecture of the project in a modern way if you agree.

markt-asf commented:
Please provide an updated screenshot showing the results of all your changes as there have been additional changes since the PR was opened.

markt-asf commented on webapps/host-manager/css/manager.css:
```diff
@@ -125,6 +132,16 @@ TD {
     font-family   : sans-serif, Tahoma, Arial;
     color         : black;
+    padding       : 5px;
+    color         : #636363;
```
Switching from Black to Grey reduces the contrast and makes it harder to read. I think it would be better left as black. It is also odd to have the titles stand out less than the content.

markt-asf commented on webapps/host-manager/css/manager.css:
```diff
@@ -138,4 +155,59 @@ form.inline {
+    box-shadow: rgba(225, 228, 232, 0.2) 0 1px 0 inset;
+    transition: none 0s;
+}
+  
```
You have trailing spaces on this line that need to be removed. If you run the build with Checkstyle enabled it will find any issues like this.

markt-asf commented on webapps/manager/css/manager.css:
```diff
@@ -125,6 +132,16 @@ TD {
     font-family   : sans-serif, Tahoma, Arial;
     color         : black;
+    padding       : 5px;
+    color         : #636363;
```
Same as above.

markt-asf commented on webapps/manager/css/manager.css:
```diff
@@ -138,4 +155,59 @@ form.inline {
+    box-shadow: rgba(225, 228, 232, 0.2) 0 1px 0 inset;
+    transition: none 0s;
+}
+  
```
Same as above

markt-asf commented:
Closing as this is a cosmetic change and requested updates to the PR have not been made in ~6 weeks.
